packages/eventual-send/src/handled-promise.js

```diff
@@ -45,6 +45,11 @@ const coerceToObjectProperty = specimen => {
   return String(specimen);
 };
 
+const hardenVoid = obj => {
+  // Do not return the hardened value
+  harden(obj);
+};
```

Comment:
It would amuse me to use `void` for its purpose just once.

```suggestion
const hardenVoid = obj => void(harden(obj));
```

Reply:
I considered it, but decided it was too obscure for reader.